.gitignore

```diff
@@ -42,3 +42,5 @@ local.properties
 # Sandbox stuff
 _sandbox
 
+signing.properties
+droidconit.jks
```

Comment:
Damn I'm on the phone and can't attach a magic.gif

On Sat, 4 Apr 2015 21:45 Eugenio Marletti <notifications@github.com> wrote:

> In .gitignore
> <https://github.com/rock3r/iosched/pull/13#discussion_r27771488>:
>
> > @@ -42,3 +42,5 @@ local.properties
> >  # Sandbox stuff
> >  _sandbox
> >
> > +signing.properties
> > +droidconit.jks
>
> @tiwiz <https://github.com/tiwiz> for the keystore, put it in a folder,
> then inside that folder put a .gitignore that ignores everything (just a *),
> so also itself... [image: :sparkles:] magic [image: :sparkles:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rock3r/iosched/pull/13/files#r27771488>.
>